surveys/sphinx-survey-2025.yml

```diff
@@ -82,14 +70,12 @@ questions:
   other: true
   choices:
   - "Academia"
-  - "Analytics / Visualisation"
-  - "Data Science"
+  - "Data Science / Analytics / Visualisation"
```

Comment:
I think vis is distinct enough that it should not be grouped with DS/Analytics. There are a lot of folks that do vis as part of other disciplines like journalism for example

Reply:
I can separate them again. I do know people may not do everything in a category, but I wasn't aware of us needing to know that difference in population for the purpose of understanding Sphinx usage at a high level.